_pytest/debugging.py

```diff
@@ -85,6 +85,8 @@ def _enter_pdb(node, excinfo, rep):
     # for not completely clear reasons.
     tw = node.config.pluginmanager.getplugin("terminalreporter")._tw
     tw.line()
+    tw.sep(">", "captured stdout")
```

Comment:
Perhaps we should print this only if there's any actual stdout:

```python
stdout = rep.capstdout
if stdout:
    tw.sep(">", "captured stdout")
    tw.line(stdout)
```

And maybe we should also print `stderr`?

Reply:
both sound good